app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update]
+
+  def index
+    @users = User.all
+  end
+
+  def show
+    @cards = current_user.cards.order('created_at')
+    @total_cards = @cards.size
+    @latest_card = @cards.last
+    @liked = current_user.find_voted_items.sort_by { |like| like.created_at }
```

Comment:
So I was thinking about having this much of instance variables and passing them all into the views where we could just call those methods on the variables in view. 
did some googling against instance variables and learned something new. Have a look at this answer, it makes a lot of sense to me http://stackoverflow.com/questions/2503838/rails-should-partials-be-aware-of-instance-variables#answer-13752155